src/components/structures/MessagePanel.tsx

```diff
@@ -461,6 +462,13 @@ export default class MessagePanel extends React.Component<IProps, IState> {
             return false; // no tile = no show
         }
 
+        if (
+            POLL_START_EVENT_TYPE.matches(mxEv.getType()) &&
+            !SettingsStore.getValue("feature_polls")
+        ) {
+            return false;
+        }
```

Comment:
this should probably be within `haveTileForEvent`

Reply:
Moved into getHandlerTile in a9be10c039